### PR TITLE
Timer fixes

### PR DESCRIPTION
**Pull request by JimCownie**
Enhancements to add the ability to run some of the load/store measurements across all combinations of source and target.
Minor tweak to `plot.py` to split the title of a plot over two lines if it is too long for one.

### CODE REVIEW

**Comment by mjklemm** [comment]
Can you please include a summary of the changes of the TimerFixes branch in this pull request?

**Comment by JimCownie** [comment]
I just pushed a commit which fixed one minor issue (missing info on the help in atomics.cc), and added a more general description of the changes. Is that what you wanted?

**Comment by mjklemm** [comment]
I was talking about CHANGELOG.md

File: microBM/atomics.cc [hunk 1]
```diff
@@ -360,8 +360,10 @@ void measureBackoff(lomp::statistic * stats) {
 
 static void printHelp() {
   printf("The first argument determines the test.\n"
-         "I[if]        -- Atomic increment of integer or float\n"
-         "B            -- Backof stats for fp add\n");
+         "I[ifet]      -- Atomic increment of integer or float\n"
```
Comment (mjklemm) [inline microBM/atomics.cc:363]:
```suggestion
         "I[ifet]      -- Atomic increment of integer (i) or float (f)\n"
```

**Comment by mjklemm** [comment]
There's only a very minor change that I have suggested?  Can you accept it into the PR, please?

**Review by mjklemm — CHANGES_REQUESTED** [review]

**Pull request merged**